config_template/test_param_config_template.h

```diff
@@ -28,6 +28,14 @@
 #ifndef TEST_PARAM_CONFIG_H
 #define TEST_PARAM_CONFIG_H
 
+/**
+ * @brief Backward compatible for coreMQTT version
+ * @note Define this if your coreMQTT version is less than or equal to v1.2.0.
+ *       Get it from coreMQTT/source/include/core_mqtt.h
+ *
+ * #define MQTT_TEST_USE_CORE_MQTT_V_1_2_0
```

Comment:
Can we use the test versioning instead of using macros to check compatibility for previous versions

Reply:
Macros removed!